### PR TITLE
Added arrow section on the top

### PR DESCRIPTION
**Pull request by jfidiles**

### CODE REVIEW

File: _sass/main/_section.scss [hunk 1]
```diff
@@ -33,6 +33,13 @@
 	padding-right: 0;
 }
 
+.Section--withTopArrow {
+	clip-path: polygon(0 0, calc(50% - #{$arrow-section-size}) 0, 50% #{$arrow-section-size}, calc(50% + #{$arrow-section-size}) 0, 100% 0, 100% 25%, 100% 90%, 100% 100%, 75% 100%, 10% 100%, 0 100%, 0% 74%);
+	-webkit-clip-path: polygon(0 0, calc(50% - #{$arrow-section-size}) 0, 50% #{$arrow-section-size}, calc(50% + #{$arrow-section-size}) 0, 100% 0, 100% 25%, 100% 90%, 100% 100%, 75% 100%, 10% 100%, 0 100%, 0% 74%);
```
Comment (peduarte) [inline _sass/main/_section.scss]: Do you need `-webkit`? Think it's cool if we only support modern browsers and I think latest versions of chrome already support this without the prefix.

File: _sass/main/_photo-section.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .PhotoSection {
 	height: $image-height-small;
 	width: 100%;
+	z-index: 0;
```
Comment (peduarte) [inline _sass/main/_photo-section.scss]: how come you need `z-index: 0`?

File: _sass/main/_section.scss (hunk 1 above)
Comment (jfidiles) [inline _sass/main/_section.scss]: It works just fine on Chrome but not on Safari, so apple users who are using Safari for mobile devices will not be able to see them.

Comment (peduarte) [inline _sass/main/_section.scss]: Cool. Let's leave it then.

**Pull request merged**